coin/coins.json

```diff
@@ -307,5 +307,12 @@
 		"symbol": "WAVES",
 		"name": "Waves",
 		"link": "https://wavesplatform.com/"
+	},
+	{
+		"index": 118,
+		"hex": "0x80000076",
+		"symbol": "ATOM",
+		"name": "Cosmos ATOM",
+		"link": "https://cosmos.network/"
```

Comment:
Needs number of decimals

Reply:
I'll find this out - this is X where (sats ÷ (10^X)) = value, right?